packages/text-field/index.js

```diff
@@ -262,7 +263,7 @@ class TextField extends React.Component {
         key='text-field-container'
       >
         {leadingIcon ? this.renderIcon(leadingIcon) : null}
-        {this.renderInput()}
+        {this.foundation_ ? this.renderInput() : null}
```

Comment:
This actually fixes https://github.com/material-components/material-components-web-react/issues/303